docs/documentation/cn/sql-reference/sql-statements/Data Definition/ddl_stmt.md

```diff
@@ -1110,44 +1110,3 @@
 ## keyword
 
     COLOCATE, JOIN, CREATE TABLE
-
-# CREATE FUNCTION
```

Comment:
Why do you delete it?

Reply:
Because there already two md file with duplicate topics named "create-function" and "drop-function" exists. The two documents originally belonged to documentation folder.